components/FormWizard/DynamicInputMulti.jsx

```diff
@@ -65,7 +67,12 @@ const DynamicInputMulti = ({
       ))}
       <div className="govuk-!-margin-top-3 govuk-!-margin-bottom-5">
         <button
-          className="govuk-link"
+          className={
+            buttonStyle === 'button'
```

Comment:
This confuses me slightly.

We set a `buttonStyle` prop equal to 'button' within the className property object, but the type is also set to "button" on line 75, all within a button component. 

I might also not understand these lines of code well, so happy for things to stay if we find that we need them. We could refactor, e.g. rename stuff to make it more straightforward 🤔